test/test.mjs

```diff
@@ -0,0 +1,27 @@
+/* set by mocha when running the test harness */
```

Comment:
The actual test file run by mocha. `describe` and `it` are only defined when running in the harness - they're meant to help describe what is being tested.

Reply:
Right now this only test the overall conversion process; a more useful test suite would probably test more of individual functions of the library.